### PR TITLE
Few docstring updates in prep for `v0.22.0`

### PR DESCRIPTION
**Pull request by antalszava**

### CODE REVIEW

File: pennylane/transforms/__init__.py [hunk 1]
```diff
@@ -112,6 +112,7 @@
 .. autosummary::
     :toctree: api
 
+    ~transforms.qcut.contract_tensors
```
Comment (trbromley) [inline pennylane/transforms/__init__.py:115]:
```suggestion
```
Thanks for spotting @antalszava! Though we did intend to not mention this function.

File: pennylane/transforms/qcut.py [hunk 2]
```diff
@@ -714,7 +714,7 @@ def contract_tensors(
 
     The network can then be contracted using:
 
-    >>> qml.transforms.contract_tensors(tensors, graph, prep, meas)
+    >>> qml.transforms.qcut.contract_tensors(tensors, graph, prep, meas)
```
Comment (trbromley) [inline pennylane/transforms/qcut.py:717]: 👍

**Review by albi3ro — APPROVED** [review]
Seems good :)

**Comment by antalszava** [comment]
Overriding the Codecov check. The coverage shouldn't have changed because the changes are only a) documentation changes b) renaming of a test file that is still being picked up by the suite.

**Pull request merged**